addons/stock/models/stock_picking.py

```diff
@@ -710,7 +710,6 @@ def action_assign(self):
         package_level_done.write({'is_done': False})
         moves._action_assign()
         package_level_done.write({'is_done': True})
-
```

Comment:
small useless diff and we are good to go

Reply:
oops.... removed